File: BennuLib.Sharp/Pixels/IPixel.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
using System.Threading.Tasks;
namespace BennuLib
{
	public interface IPixel
	{
		int Argb { get; }
		int Red { get; }
		int Green { get; }
		int Blue { get; }
		int Alpha { get; }
		int Value { get; }
		IPixel GetTransparentCopy();
		IPixel GetOpaqueCopy();
		bool IsTransparent { get; }
	}
}

File: BennuLib/Pixels/IPixel.vb
Public Interface IPixel
    ReadOnly Property Argb() As Integer
    ReadOnly Property Red As Integer
    ReadOnly Property Green As Integer
    ReadOnly Property Blue As Integer
    ReadOnly Property Alpha As Integer
    ReadOnly Property Value As Integer
    Function GetTransparentCopy() As IPixel
    Function GetOpaqueCopy() As IPixel
    ReadOnly Property IsTransparent As Boolean
End Interface